examples/vic-gov-au/test/e2e/integration/core-modules/landingPage/LandingPage.feature

```diff
@@ -1,23 +1,203 @@
+      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Aerial%20shot%20of%20new%20housing%20development.jpg | 2 / 4 - Demo: Aerial shot of new housing development |
+      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Bendigo%20Hospital.jpg                               | 3 / 4 - Demo: Bendigo Hospital                       |
+      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Melbourne%20tram.jpg                                 | 4 / 4 - Demo: Melbourne tram                         |
+
+    # Card Event Automated
+    And there should be an event card with the title "Demo Event"
+    And the event card titled "Demo Event" should contain the following:
+      | date          | title      | summary                                                                                                                                            | address   | link        | linktext   |
+      | 02 to 07 July | Demo Event | Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam tincidunt sit amet ligula sit amet lacinia. In a leo nec tortor aliquet faucibus. | Melbourne | /demo-event | View  more |
+
+    # Card Carousel - add card event
+    And there should be a card carousel titled "Card Carousel for Card Event"
+    And the card carousel titled "Card Carousel for Card Event" should have 2 items
+    And the card carousel titled "Card Carousel for Card Event" should have the following items:
+      | date                           | title                 | image                                                                                                          | summary                                                                                     | address   | link                   | linktext   |
+      | 01 January to 12 December 2020 | Carousel Card Event   | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Melbourne%20tram.jpg | Mauris tincidunt tincidunt felis vel tempus. Vestibulum rhoncus blandit justo quis finibus. | Melbourne | https://www.vic.gov.au | Read  more |
+      | 01 January                     | Carousel Card Event 2 | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Melbourne%20tram.jpg | Mauris tincidunt tincidunt felis vel tempus. Vestibulum rhoncus blandit justo quis finibus. | Melbourne | https://www.vic.gov.au | Read  more |
+
+    #  Latest Events
+    And there should be a latest events titled "Demo latest events"
+    And the latest events titled "Demo latest events" should have 6 items
+    # Disabled as Latest Events data is not stable.
+    # And the latest events titled "Demo latest events" should have the following items:
```

Comment:
Yeah we need to have a BE we can curate the content on to be able to do this kind of test